### PR TITLE
Add minutes from 2015-04-16 meeting

### PR DESCRIPTION
**Pull request by srl295**
Minutes..

### CODE REVIEW

**Comment by srl295** [comment]
@joyent/node-coreteam FYI

**Comment by mhdawson** [comment]
The github link for me should be https://github.com/mhdawson

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+JGI: always possible for users to specify custom cipher lists. 
+
+TJ: balance between platform users and application users
+
+JGI: need to get the main scenarios, come up with use cases first
+
+MD: this is a tempoary fix, most people (hopefully) are using defaults.
+
+TJ: this is more in minor versions, when peopel are bringing up the app
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: `peopel` -> `people`. Although I'm not sure what the whole sentence refers to.

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+	** will look into the wrong checksums
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: I believe this belongs to a separate topic (wrong checksum for release files for node v0.12.2).

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+TJ: this should be first class documentation.
+
+- [ ] srl, open a new ticket. Document that Date.now  != how timers work. (doc/timers)
+
+### openssl
+
+JGI: openssl upgrades, confusing
+
+* often these upgrades are done in 'less than ideal' situations
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md:87]: I don't think we need a bullet point here, and we might want to connect this part of the sentence with the previous one.

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+- [ ] SRL: action to document how to upgrade ICU
+
+### Joint node.js/io.js
+
+CI: joint TC call
+
+* People don't want extra meetings - may be a little disorganized
+
+TJ: - hard but necessry- there's a lto of work to do around merging two projects
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: `necessry` -> `necessary`.

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+* need to work together to maintain stable release (LTS/platform etc) has to be comfitable integrating into the entire project
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: `comfitable` -> `comfortable`? If possible, let's make sure that all lines wraps at under 80 columns too.

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+SRL: I wasn't able to relabel tickets
+
+TJ: will fix (fixed)
+
+### Crash from command line (Intl)
+
+* crash from command line? [#9361](https://github.com/joyent/node/issues/9361)
+
+ * (SRL to investigate)
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: Probably unnecessary sub bullet point.

Comment (srl295) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: removed

**Comment by srl295** [comment]
Review notes incorporated

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+TJ:  blog post , link to documentation/faq
+
+TJ / AC:clock used on windows for 0.10 is less precise but still monotonic
+
+MD: google response on v8 - "thats a good qeustion"
+
+TJ: this should be first class documentation.
+
+- [ ] srl, open a new ticket. Document that Date.now  != how timers work.
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: Maybe we could replace `[ ]` with: `Action item:...`?

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+* ideally anyone on the team or community can understand how it is done
+
+TJ: no specific home for this doc, in the wiki right now (link)
+
+* also one on v8 - may not include all floating patches
+
+TJ:   [docs] should live in the code eventually
+JGI:  may take another form. 
+
+- [ ] SRL: action to document how to upgrade ICU
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: Maybe we could replace `[ ]` with: `Action item:...`?

File: doc/about/core-team/meetings/2015-04-16/minutes.md (hunk 6 above)
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: Did you push the removal? It seems I can still see it.

Comment (srl295) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: I see it as gone, but it took some time before it appeared on GH

Comment (srl295) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: sorry! I removed the wrong one. fixed

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+JGI: need to get the main scenarios, come up with use cases first
+
+MD: this is a tempoary fix, most people (hopefully) are using defaults.
+
+TJ: this [setting] is [used] more in minor versions, when people are
+bringing up [some version of an] app.
+
+#### Wrong checksum for node v0.12.2
+
+* This issue will be investigated. (by whom?)
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: No need for bullet point, issue was investigated by me.

File: doc/about/core-team/meetings/2015-04-16/minutes.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+JGI:  maybe, :15 joint meeting [ then individual meetings ]
+
+### Call for help triaging incoming bugs
+
+JGI: Help wanted triaging incoming bugs
+
+### Crash from command line (Intl)
+
+* crash from command line? [#9361](https://github.com/joyent/node/issues/9361)
```
Comment (misterdjules) [inline doc/about/core-team/meetings/2015-04-16/minutes.md]: No need for bullet points.

**Comment by misterdjules** [comment]
@srl295 Thank you! Please squash all the commits in one commit that has "Add minutes from 2015-04-16 meeting" as a commit message. Once that is done, LGTM!

**Comment by srl295** [comment]
@misterdjules thank you!

**Comment by misterdjules** [comment]
LGTM, thank you @srl295.

**Comment by misterdjules** [comment]
Landed in 90ad03c29a0845d02ad4db6c2d1b8517ed820e79, thanks again!

**Comment by misterdjules** [comment]
Deployed at https://nodejs.org/about/core-team/meetings/2015-04-16/minutes.html.

**Pull request closed without merging**